src/_color-use-cases.scss

```diff
@@ -1 +1 @@
-@include oColorsSetUseCase(o-quote, background, 'black-5');
```

Comment:
i'm not sure if deleting a use case counts as a breaking change?

Reply:
I'd think technically yes, but I can't see any existing way to mark a single property `background` as deprecated. Seems the only way is to deprecate `o-quote` as a whole usecase for all properties, background, border, and otherwise. But don't trust a newbie: Maybe someone else (@Financial-Times/origami-core) knows if there's an existing precedent for this.

Reply:
I think possible options are:
* Keep the background as is, but just don't use it
* Keep the background use case as `transparent`, and use it for setting the transparent background

Reply:
I'd deffo go option 2, I didn't realise that `transparent` was a valid palette color but it is 👍